libs/ballot-interpreter/src/adjudication_reasons.ts

```diff
@@ -7,10 +7,31 @@ import {
   MarkStatus,
   WriteInAreaStatus,
 } from '@votingworks/types';
-import { find, throwIllegalValue } from '@votingworks/basics';
+import { assertDefined, throwIllegalValue } from '@votingworks/basics';
 import { allContestOptions } from '@votingworks/utils';
 import { type ScoredContestOption } from './legacy_adapter';
 
+function rankMarkStatus(markStatus: MarkStatus): number {
+  switch (markStatus) {
+    case MarkStatus.Marked:
+      return 2;
+    case MarkStatus.Marginal:
+      return 1;
+    case MarkStatus.Unmarked:
+      return 0;
+    /* istanbul ignore next */
+    default:
+      throwIllegalValue(markStatus);
+  }
+}
+
+function sortMarkStatusDescending(
```

Comment:
nit: I'd name this `compareMarkStatusDescending` since it's not actually sorting anything.